docs-source/docs/modules/ROOT/pages/index.adoc

```diff
@@ -4,24 +4,19 @@
 include::partial$include.adoc[]
 
 
-=== What is this guide about?
-This guide provides information to aid in understanding and using the Akka Platform and the Akka Cloud Platform.
+== What is this guide about?
+This guide provides information to aid in understanding and using the Apache Pekko.
```

Comment:
remove `the` before Apache Pekko - this does not look correct to a native English speaker like me - you can say 'the Apache Pekko Platform' but without 'Platform', English speakers would remove the 'the'.